client/google-search-app/src/app/services/search.service.ts

```diff
@@ -0,0 +1,56 @@
+import { Injectable } from '@angular/core';
+import { SearchResults } from '../model/search-results';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SearchService {
+  constructor() {}
+
+  getSearchResults(query: string): SearchResults {
+    // API call logic goes here
+    return {
+      total_results: 6,
```

Comment:
instead of hard coding directly. you can define the method / variable called 

```
const monkeyPatch = { ... }
or
const guerrillaPatch = { ... }
```
```

Reply:
Need more information on it. I am not getting it.